server/routes/visitors.test.ts

```diff
@@ -201,33 +201,22 @@ describe('POST /visit/select-visitors/A1234BC', () => {
         expect(res.text).toContain('28 July 1986<br>(Adult)')
         expect(res.text).toContain('Sister')
         expect(res.text).toContain('123 The Street')
-        expect(res.text).toContain('visitor-restriction-badge--BAN">Banned</span> until 31 July 2022')
```

Comment:
These assertions were incorrectly passing. Fixed by explicitly handling `err` and calling `done()` in `end()`. Took me a while to figure out what was going on here!